### DISCUSSION TITLE
一个改进提议

### DISCUSSION THREAD

**Discussion by Elviserum** [zhongyang219/TrafficMonitor#1482] [Ideas]
一个提议
能否把软件增加一个exe安装版本，每次更新都是下载一个新的zip包然后覆盖，有点点麻烦

**Comment by soloopooo** [comment]
相比而言，我觉得与其他软件类似 都放出 portable 和 安装版 两个版本会更好一些

**Reply by Elviserum** [reply]
> 相比而言，我觉得与其他软件类似 都放出 portable 和 安装版 两个版本会更好一些

对的，就是这个意思，我表达的有点急了
